src/main/kotlin/com/exactpro/th2/httpserver/api/impl/BasicResponseManager.kt

```diff
@@ -36,10 +33,14 @@ import java.util.UUID
 class BasicResponseManager : IResponseManager {
 
     private val logger = KotlinLogging.logger {}
+    private val dialogs = HashMap<String,RequestData>()
```

Comment:
Use concurrent map instead because `handleRequest` / `handleResponse` probably will be called from different threads